doc/release-notes/0.18/0.18.md

```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2019, 2019 IBM Corp. and others
+* Copyright (c) 2020, 2020 IBM Corp. and others
```

Comment:
shouldn't this be 2019, 2020 ?

Reply:
Yes probably, as we had something published for MS1. I was fixated on it being a January release. Will change.